Sources/Roboflow/Classes/Roboflow.swift

```diff
@@ -154,8 +170,20 @@ public class RoboflowMobile: NSObject {
             }
             
             let colors = coreMLDict["colors"] as? [String: String]
-            let classes = coreMLDict["classes"] as? [String]
-            
+            var classes = coreMLDict["classes"] as? [String]
+
+            let environment = coreMLDict["environment"] as? String
+
+            // download json file at the url in `environment`
+            let environmentURL = URL(string: environment!)
+            let environmentData = try? Data(contentsOf: environmentURL!)
+            let environmentDict = try? JSONSerialization.jsonObject(with: environmentData!, options: []) as? [String: Any]
```

Comment:
this dict shoudl have info on if it needs softmax or sigmoid or none

Reply:
I dont see anything that jumps out as specifying whether it needs softmax, sigmoid, or none in the environment dict

Reply:
multiclass only appears in a multiclass model, resolved in latest commit